src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/personalrelationships/profiledetails/ContactPersonProfileDetailsSyncIntTest.kt

```diff
@@ -314,7 +377,8 @@ class ContactPersonProfileDetailsSyncIntTest(
       eq("contact-person-${profileType.identifier}-$telemetryType"),
       check {
         assertThat(it["offenderNo"]).isEqualTo(offenderNo)
-        assertThat(it["dpsId"]).isEqualTo(dpsId.toString())
+        assertThat(it["requestedDpsId"]).isEqualTo(requestedDpsId.toString())
+        dpsId?.run { assertThat(it["dpsId"]).isEqualTo(dpsId.toString()) }
```

Comment:
I realised that the dpsId sent in the event might not be the same as the latest dpsId when calling the DPS endpoint. So I've split these out into different telemetry properties.